src/types/types.ts

```diff
@@ -760,6 +760,22 @@ export interface GetFloorPriceResponse {
   readonly looksRare: FloorPriceMarketplace | FloorPriceError;
 }
 
+/**
+ * Information about the rarity of an NFT's attribute in the specified collection.
+ *
+ * @public
+ */
+export interface NftAttributeRarity {
+  /** Name of the NFT's attribute. */
+  value: string;
+
+  /** The type of NFT attribute. */
+  trait_type: string;
+
+  /** The NFT's attribute frequency in the current collection. */
```

Comment:
Please update comment to:
`A number from 0 to 1 representing the prevalence of this value for this trait type in the current collection.`

Reply:
Okis, much better